_includes/page-highlights.html

```diff
@@ -14,7 +14,11 @@
     <p class="page__excerpt">
       {{ page.excerpt }}
     </p>
-    {%- include img-caption.html tag='p' desc=site.data.language.photo_credit source=page.image_credit -%}
+    {% if page.name == 'about.md' or page.name == 'events.md' or page.name == 'members.md' %}
+      {%- include img-caption.html tag='p' desc=site.data.language.photo_credit header_caption=page.header_image_caption source=page.header_image_credit header=true -%}
```

Comment:
This it the right idea, but let's match the style that exists on inline posts. So instead of having both the image caption and "Photo Credit", you can replace the `desc` parameter with page.header_image_caption. Notice in the picture below, there is no explicit "Photo Credit" text. Instead, the source is differentiated by having a different color.

<img width="484" alt="screen shot 2018-09-28 at 11 06 47 am" src="https://user-images.githubusercontent.com/1896496/46216920-a29da900-c30e-11e8-838d-0f730d0a1d29.png">

Reply:
Changes have been made.